src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HttpConnection.cs

```diff
@@ -723,7 +746,7 @@ private CancellationTokenRegistration RegisterCancellation(CancellationToken can
             }, _weakThisRef);
         }
 
-        private static bool LineIsEmpty(ArraySegment<byte> line) => line.Count == 0;
+        private static bool IsLineEmpty(ArraySegment<byte> line) => line.Count == 0;
```

Comment:
Why did you rename this?  FWIW, I find it reads much more naturally as "if line is empty" rather than "if is line empty".

Reply:
I got suggestion from code review in private branch: Name should be verb-noun for things that return values such as bool checking for "existence". I don't have strong preference, but I think we should have a standard.

cc:/ @davidsh